### DISCUSSION TITLE
Use ``if:`` for diferent actions

### DISCUSSION THREAD

**Discussion by Vctrsnts** [Flexget/Flexget#3707] [Q&A]
Hi

I have a many tasks ( download podcast ) that make the same, but the unique diference is the origen:

```
task_1:
 rss: http://feed_1.xml
 download
 move to directory one
task_2:
 rss: http://feed_2.xml
 download
 move to directory two
task_3:
 rss: http://feed_3.xml
 download
 move to directory three
```

And now, trying to change this *same* task for unique task with differents action when *author* if different:
```
task:
 input:
  - rss: http://feed_1.xml
  - rss: http://feed_2.xml
  - rss: http://feed_3.xml
 if:
  - "'author_1' in author":
    download
    move to directory one
  - "'author_2' in author":
    download
    move to directory two
  - "'author_3' in author":
    download
    move to directory three
```

Is possible this??

Thanks a lot

**Comment by BrutuZ** [comment]
Assuming you're just setting the paths for the download and move, it is :+1:
Could've just tried it with the [`--test`](https://flexget.com/CLI#flexget-level-arguments) flag too (`flexget --test execute --task task`), would probably have been quicker :P

**Comment by Vctrsnts** [comment]
Sorry,

I'm sorry but I don't understand what you mean by using the ``--test`` or the path or the move??

**Reply by BrutuZ** [reply]
`--test` is a command-line flag as described in the link. It will create a copy of the database when you execute the task so that you can try without affecting further runs, most notably entries being rejected by the `seen` plugin because they were preciously accepted\rejected on a trial run.

You can set the `{{path}}` field (https://flexget.com/Plugins/if#set-field-values) that will be used by the `move` plugin and some output plugins like `download` (https://flexget.com/Plugins/set#plugins-that-accept-set-keywords)

**Comment by Vctrsnts** [comment]
Thank you so much.

``--test`` I use it a lot but I didn't quite understand what you wanted to say.

I have already done tests with the code, which I have at the moment, and the messages that appear to me is that the ``if`` only accepts ``accept`` or ``reject``.

With respect to the second point, as I understand it, it is to indicate the path, so that the download goes directly to the directory in question, what worries me is all the treatment that I do afterwards, renaming mp3, tags, etc...

Thank you
